conf/application.conf

```diff
@@ -80,7 +80,7 @@ platform {
   }
 }
 
-play.filters.headers.contentSecurityPolicy = "default-src 'unsafe-inline' 'self' localhost:9032 data: https://www.google-analytics.com https://www.googletagmanager.com; script-src 'unsafe-inline' 'self' localhost:9032 https://www.google-analytics.com data: https://www.googletagmanager.com https://tagmanager.google.com; font-src 'self' data: https://ssl.gstatic.com https://www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com; img-src 'self' localhost:9032 https://ssl.gstatic.com https://www.gstatic.com https://www.google-analytics.com https://www.googletagmanager.com data:; style-src 'self' 'unsafe-inline' localhost:9032 https://tagmanager.google.com https://fonts.googleapis.com"
+play.filters.headers.contentSecurityPolicy = "default-src 'unsafe-inline' 'self' localhost:9032 data: https://www.google-analytics.com https://www.googletagmanager.com; script-src 'unsafe-inline' 'self' localhost:9032 https://www.google-analytics.com data: https://www.googletagmanager.com https://tagmanager.google.com; font-src 'self' data: https://ssl.gstatic.com https://www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com; img-src 'self' https://ssl.gstatic.com https://www.gstatic.com https://www.google-analytics.com https://www.googletagmanager.com data:; style-src 'self' 'unsafe-inline' https://tagmanager.google.com https://fonts.googleapis.com"
```

Comment:
I *think* this should be the CSP on the left! I think for some reason this is removing two additional localhost rules. Not sure why it was changed in the branch (did we merge from master or something?)

Reply:
There was a merge from master.  Did some of the CSP corrections get done in this branch ?

Reply:
Sorted. I believe